packages/api/src/learning-materials/learning-materials.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Get, Post, UseGuards } from '@nestjs/common';
+
+import type {
+  CreateLearningMaterialResponse,
+  GetLearningMaterialResponse,
+} from '@coderscamp/shared/models/learning-material';
+
+import { JwtAuthGuard } from '../auth/jwt/jwt-auth.guard';
+import { UserId } from '../auth/jwt/user-id.decorator';
+import { LearningMaterialsService } from './learning-materials.service';
+
+@UseGuards(JwtAuthGuard)
+@Controller('learning-materials')
+export class LearningMaterialsController {
+  constructor(private readonly learningMaterialsService: LearningMaterialsService) {}
+
+  @Get()
+  getLearningMaterial(@UserId() userId: number): Promise<GetLearningMaterialResponse> {
```

Comment:
Im not sure if it's LearningMaterial or Materials. Or maybe we should call it LearningMaterialsList?

Reply:
I think LearningMaterial could be a good name as it allows us to potentially create more types of materials in the future. In such a case LearningMaterials would be reserved for an array o LearningMaterial elements